soap/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for soap
 // Project: https://www.npmjs.com/package/soap
-// Definitions by: Nicole Wang <https://github.com/nicoleWjie>, Cage Fox <https://github.com/cagefox>
+// Definitions by: Leo Liang <https://github.com/aleung>, Cage Fox <https://github.com/cagefox>
```

Comment:
See my previous comment:

> @nicoleWjie Per your request, I change your name with mine from the 'Definitions by' list.

She is my team mate. We use node-soap in our project. She doesn't want to maintain this definition any more and let me do it.